### PR TITLE
search by letters from ps.com

### PR DESCRIPTION
**Pull request by tr211**

### CODE REVIEW

File: fastAPI.py [hunk 1]
```diff
@@ -12,10 +13,23 @@ def country_prices(game_name)->list[ps_store_repository.Game]:
...
+
 @app.get('/ps4-names')
-def ps4_games_names()->list[str]:
+def ps4_games_names() -> list[str]:
     names_list = names_registry.get_all_ps4_games()
-    return names_list 
+    return names_list
+
+
+# @app.get('/test/{game}')
```
Comment (radprofy) [inline fastAPI.py:27]: Do not leave trash in the code.

File: globe_search.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.service import Service as ChromeService
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.chrome.options import Options
+import time
+
+def searcher_name_games(game: str) -> list:
+    game_html_format = game.replace(' ', '+')
+    search_game = game_html_format.lower()
```
Comment (radprofy) [inline globe_search.py:10]: You can do that without introducing any new variables
game = game.replace().lower()...

File: globe_search.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    # Setup Selenium WebDriver
+    options = Options()
+    options.headless = True  # Run in headless mode
+    driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()), options=options)
+
+    url = f'https://www.playstation.com/en-us/search/?q={search_game}&category=games'
+    driver.get(url)
+    
+    # Wait for the page to load
+    time.sleep(5)
```
Comment (radprofy) [inline globe_search.py:24]: Static delays are very bad, is there any way to wait for element?

File: fastAPI.py (hunk 1 above)
Comment (tr211) [reply fastAPI.py:27]: made clear

File: globe_search.py (hunk 2 above)
Comment (tr211) [reply globe_search.py:10]: solved

File: globe_search.py (hunk 3 above)
Comment (tr211) [reply globe_search.py]: remove for asyncio

**Pull request closed without merging**